canopy_main.ml

```diff
@@ -61,9 +61,9 @@ module Main  (C: CONSOLE) (S: STACKV4) (RES: Resolver_lwt.S) (CON: Conduit_mirag
     store_ops.Canopy_dispatch.update () >>= fun l ->
```

Comment:
We can use only `update_atom ()` but we cannot follow with `Lwt_list.iter_p (C.log_s console) ` 
anymore, as this uses the result of `Store.fill_cache`... but maybe that's OK ?